### ISSUE TITLE
Request: single mouse click to toggle play/pause

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [Stremio/stremio-shell#101]
on the content itself.

**Comment by TRtomasz** [comment]
I'v checked VLC and Netflix players they don't have such feature. Only player i know about is Youtube and it is the most annoying feature there because it interfere with double click to change window size.

**Comment by AndroidDeveloperLB** [comment]
That's incorrect. Many players do it, including on Android.
MPC has it for example.
Please at least have it as an option.
And why do you say it's annoying? Currently nothing is assigned to single click.

**Comment by Ivshti** [comment]
hey,

we may introduce an option for this but it won't become default, since almost no desktop apps do it, so it's not a behavior users are used to

on mobile/web, it's different

**Comment by AndroidDeveloperLB** [comment]
Almost none? How many have you tried that you say it?
Almost all.
And why would you want it to be different on mobile/web? Why not have consistent experience and easy to use like on the mobile/web? 
You've shown only 2 video players. Here are more, that all have this function, built in : 

1. MPC
2. project butter
3. Popcorn time (similar to Stremio)
4. ZoomPlayer
5. PotPlayer (Daum player) - has option, but disabled by default.
6. KMPlayer - has option, but disabled by default.
7. SMPlayer   - has option, but disabled by default.
8. GOM player - has option, but disabled by default.

And I'm sure there are more. And that's just on Windows.

I remember even the old Crystal player had it.

It's a very commonly used feature and very familiar, on both PC, web and Android. 

And you didn't explain why it shouldn't be added and be the default behavior.
It's not taken by any other action. It doesn't do anything on Stremio, but for some reason double clicking So why not?

**Comment by aluskler** [comment]
how do i use this

**Comment by AndroidDeveloperLB** [comment]
Sadly this was marked as closed as if it was implemented, but it wasn't.
I don't understand what's wrong with having it by default. It is not used for anything else.

**Comment by GitExploraa** [comment]
having it by default would be kinda weird imo (since not a lot of people use it), but they should at least give us the option to do so

**Comment by GitExploraa** [comment]
> @GitExploraa Why would it be weird? It's not used. It's a new feature that is being used already on many video players. It's actually weird that it's not turned on by default. Users might not notice it till they reach the settings when it's turned off by default. :)
> 
> I don't understand your comment of "not a lot of people use it". Nobody uses it yet on Stremio because it doesn't exist yet. After it's turned on, I guarantee many more will use it. Maybe even more than the play button.
> 
> I also wish I could drag-press, to move the window, and also be able to right click even while it's loading (copy magnet for example in case it's slow loading, to download on a torrents app).
> 
> All of these are new. Not used, so it's safe to enable by default.

well lets first hope they add it at all, then you can go on asking for it to be enabled by default x)

**Comment by AndroidDeveloperLB** [comment]
Right. Sadly this request was closed, so please create a new request.

**Comment by christiansuryanto** [comment]
> Right. Sadly this request was closed, so I think people forgot about it

Bump..

**Comment by dav1312** [comment]
> I'v checked VLC and Netflix players they don't have such feature. Only player i know about is Youtube and it is the most annoying feature there because it interfere with double click to change window size.

VLC has a pretty popular plugin that does this because a lot of people (me included) love this functionality

https://github.com/nurupo/vlc-pause-click-plugin

**Comment by coxde** [comment]
Could devs reopen this feature request?

**Comment by kiviuk** [comment]
Any news on this?
